gcp/README.md

```diff
@@ -151,41 +151,18 @@ Workflow: We will create a notebook instance, and run a shell script that will i
   
 1. Log into your GCP console.
     1. Select AI-Platform -> Notebooks
-    1. Select a "Create new notebook". And select the RAPIDS XGBoost variant (comes with Conda installed)
+    1. Select a "New Instance" -> "Python 3 CUDA Toolkit 11.0" -> With 1 NVIDIA Tesla T4
         1. Select 'install gpu driver for me'
-        1. Select 'customize'
-            1. Pick the CUDA variant you want (10.1, 10.0, etc..)
-            1. For your GPU type, select T4, or V100
-            1. Select the number of GPUs 1-8
-        1. Launch your notebook service.
+        1. Create.
     1. Once JupyterLab is running
         1. Open a new terminal
-        1. Copy the 'rapids-py37-kernel.sh' GCP script into the local environment.
-        1. Run the script
-            1. Once completed, you will have a new kernel in your jupyter notebooks called 'rapids_py37' which will have rapids installed.
-
-### Deploy a custom RAPIDS container notebook
-Motivation: We want to build a custom docker container for the GCP AI Platform, with out of the box support for RAPIDS, and pre-installed Jupyter kernels.  
-Workflow: We will build a custom docker container with RAPIDS, using Google's recommended base image, push this container to the Google Container Registry (GCR),
-and launch an AI Platform notebook backed on this container.
-
-1. Pull or create custom container
-    1. Build locally
-        1. `$ cd cloud_service_providers/gcp/docker`
-        1. `$ docker build --tag gcr.io/[YOUR PROJECT NAME]/rapids-py38 --file Dockerfile.jupyter_notebook ./`
-1. Push to the Google Container Registry
-    1. Ensure gcloud is installed and you have configured the GCR authentication helper for Docker.
-        1. See: https://cloud.google.com/container-registry/docs/advanced-authentication 
-    1. `$ docker push gcr.io/[YOUR PROJECT NAME]/rapids-py38`
-1. Log into your GCP console.
-    1. Select AI-Platform -> Notebooks
-    1. Select a "New Instance" -> "Customize Instance"
-        1. Name your instance
-        1. Select Environment -> Custom Container
-            1. Enter: gcr.io/[YOUR PROJECT NAME]/rapids-py38 
-        1. Select 'install gpu driver for me'
-        1. Select 'customize'
-            1. For your GPU type, select T4, or V100
-            1. Select the number of GPUs 1-8
-        1. Launch your notebook service.
-    1. Once JupyterLab is running, you will have a kernel available in your jupyter notebooks called 'rapids_py38' which will have rapids installed.
+        1. Run
+           ```shell
+           RAPIDS_VER=0.18
+           CUDA_VER=11.0
+           wget -q https://rapidsai-data.s3.us-east-2.amazonaws.com/conda-pack/rapidsai/rapids${RAPIDS_VER}_cuda${CUDA_VER}_py3.8.tar.gz
```

Comment:
We need to keep this, because its the fastest way for them to get RAPIDS into their container, because they can't bring their own container with rapids pre-installed.

We don't have anything like the S3 public buckets on GCP.

Reply:
Yes we definitely need to keep it, just might come up for internal users if they don't want to pull from S3. Can modify later if there are any issues